client/src/components/WordOfTheDay/WordOfTheDay.tsx

```diff
@@ -5,9 +5,15 @@ import PostExpanded from '../PostExpanded/PostExpanded';
 import formatDate from '../../utils/formatDate';
 import wotdPostIds from '../../constants/wotdPostIds';
 import { getWordOfTheDayPost } from '../../actions/posts';
+import { Post } from '../../../types/Post';
 
+type GlobalState = {
+    posts: Post[] & {wordOfTheDayPost: Post}
+    [x:string]: any
+}
 const WordOfTheDay = () => {
-    const { wordOfTheDayPost } = useSelector(state => state.posts)
+    // TODO: This is a hack. Refactor typings.
```

Comment:
I'll add it to my agenda, might take a while to get to it though because of Finals